src/components/MintNFT/Card.tsx

```diff
@@ -154,7 +153,11 @@ const Card: React.FC<CardProps> = ({ currentStep, draft, offMarketNft}) => {
         saveNFT(nft)
         return history.push('summary')
       case 7:
-        if (auction.isForSale || auction.isForBid) {
+        const { isForSale, listingPrice, isForBid, minBid, qualifyAmount, expireTimestamp } = auction
+        if (
```

Comment:
in this case we need to add else and handle errors

Reply:
how should we handle them @ahmedgalalmohamed2016